src/components/input-field/input-field.stories.js

```diff
@@ -1,23 +1,39 @@
 import { html } from "lit-html";
 import "../input-field";
+import "../button";
+import "../link";
+import "../form-message";
 import styles from "./styles.scss";
+import buttonStyles from "../button/styles.scss";
+import linkStyles from "../link/styles.scss";
+import formMessageStyles from "../form-message/styles.scss";
 import {
   stylesheet,
   docsIntro,
   CATEGORIES,
 } from "../../../.storybook/utils.js";
+// import { args, argTypes } from "../form-validation/config";
+// import { VALIDATION_TYPE } from "../form-validation/enum";
 
 const defaultArgs = {
   block: false,
   disabled: false,
   error: false,
   small: false,
   success: false,
+  // ...args,
 };
 
 export default {
   title: "native-elements/vl-input-field",
-  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  decorators: [
+    (story) =>
+      html`
+        ${stylesheet(
+        `${styles}${buttonStyles}${linkStyles}${formMessageStyles}`
```

Comment:
Al die stylesheets lijken me niet nodig nu.

Reply:
Idd dat waren er voor de validation stuff... ik heb ze ook in comment gezet.